graph/telemetry/istio/appender/appender.go

```diff
@@ -28,6 +30,8 @@ func ParseAppenders(o graph.TelemetryOptions) (appenders []graph.Appender, final
 			// namespace appenders
 			case AggregateNodeAppenderName:
 				requestedAppenders[AggregateNodeAppenderName] = true
+			case AmbientAppenderName:
+				requestedAppenders[AmbientAppenderName] = true
```

Comment:
I missed this, please move this below to the "Finalizer" section.

Reply:
Moved with the Finalizer appenders